src/main/webapp/WEB-INF/views/EditUserPage.html

```diff
@@ -0,0 +1,190 @@
+                                             width="120px" height="120px"/>
+                                    </div>
+                                    <div class="col-md-7">
+                                        <span>Welcome back,</span>
+                                        <p class="text-muted">UserName!</p>
+                                        <div class="btn-group-vertical" style="margin-top: -30px">
+                                            <a href="#" class="btn btn-link btn-md"><i class="fa fa-user-o"
+                                                                                       aria-hidden="true"
+                                                                                       style="margin-right: 5px"></i>My
+                                                Account</a>
+                                            <a href="#" class="btn btn-link btn-md"><i class="fa fa-star"
+                                                                                       aria-hidden="true"
+                                                                                       style="margin-right: 5px"></i>Favourites</a>
+                                            <a href="#" class="btn-link btn btn-md"><i class="fa fa-question-circle-o"
+                                                                                       style="margin-right: 5px"></i>Help
+                                            </a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="navbar-footer">
+                                <div class="navbar-footer-content">
+                                    <div class="row">
+                                        <div class="col-md-6 pull-right">
+                                            <a href="#" class="btn btn-default btn-block"><i class="fa fa-sign-out"
+                                                                                             aria-hidden="true"></i>
+                                                Sign
+                                                Out</a>
+                                        </div>
+
+                                    </div>
+                                </div>
+                            </div>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+        </div>
+    </div>
+</div>
+<!--------------------------------------------------------------------------------------->
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="panel panel-success">
+                <div class="panel-heading">
+                    <h4>User Profile</h4>
+                    <a class="btn btn-edit pull-right" href="DesignPage1.html"><h4><i
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)